app/frontend/javascript/controllers/calendar_controller.js

```diff
@@ -11,6 +11,9 @@ export default class extends Controller {
   connect() {
```

Comment:
I have no idea about most parts of this JS code. I will have to study it and I would like to see it working on the web app perhaps that way I'll be able to get it easily.

Reply:
nice i'll try that. and yeah ive read the documentation, its actually very helpful.